packages/vapor/scss/components/tile.scss

```diff
@@ -0,0 +1,65 @@
+.tile {
+    position: relative;
+    top: 20px;
+    left: 20px;
+    width: 264px;
+    height: auto;
+    min-height: 221px;
+    background: var(--white);
+    border: 1px solid var(--grey-40);
+    border-radius: 8px;
+    box-shadow: 0px 2px 8px rgba(229, 232, 232, 0.75);
```

Comment:
Any css variable we can reuse from [here](https://github.com/coveo/react-vapor/blob/master/packages/vapor/scss/utility/shadow.scss#L2-L10)?

Reply:
Oh maybe! :eyes: